serverless/getting_started/installing-knative-serving.adoc

```diff
@@ -10,7 +10,7 @@ After you install the {ServerlessOperatorName}, you can install Knative Serving
 
 This guide provides information about installing Knative Serving using the default settings. However, you can configure more advanced settings in the KnativeServing custom resource definition.
 
-For more information about configuration options for the KnativeServing custom resource definition, see xref:../installing_serverless/serverless-install-config-options.adoc#serverless-install-config-options[Advanced installation configuration options].
+For more information about configuration options for the KnativeServing custom resource definition, see xref:../../serverless/serverless-install-config-options.adoc#serverless-install-config-options[Advanced installation configuration options].
```

Comment:
You have "Custom Resource Definition (CRD)" elsewhere in this PR, so I'd capitalize here for consistency.

Reply:
I think it should actually be `CustomResourceDefinition` according to https://github.com/openshift/openshift-docs/pull/22440 - I forgot about this tbh.

It doesn't seem to be consistently applied in a bunch of places across docs yet though, most seem to use "Custom Resource Definition (CRD)" or similar, so I will use this for now and put in a separate PR later to update all the instances to `CustomResourceDefinition` if that's OK?